app/grandchallenge/cases/tasks.py

```diff
@@ -509,6 +511,9 @@ def _delete_session_files(*, session_files):
                     and Path(file.filename).suffix == ".dcm"
                     and getattr(file.creator, "username", None) in users
                 ):
+                    saf.staged_files.update(
+                        timeout=timezone.now() + timedelta(days=100)
```

Comment:
```suggestion
                        timeout=timezone.now() + timedelta(days=21)
```

Less extreme, and still gives some time to pick it up after a vacation.